modules/actionagent/callback.go

```diff
@@ -76,6 +75,28 @@ func (agent *Agent) Callback() {
 	}
 }
 
+func (agent *Agent) SetCallbackReporter() {
+	tokenForBootstrap := os.Getenv(apistructs.EnvOpenapiTokenForActionBootstrap)
+	if tokenForBootstrap == "" {
+		agent.AppendError(errors.Errorf("missing env %s", apistructs.EnvOpenapiTokenForActionBootstrap))
+		return
+	}
+	agent.EasyUse.TokenForBootstrap = tokenForBootstrap
```

Comment:
split into two single methods is clearly.

Reply:
adjusted